package/boot/uboot-rockchip/src/arch/arm/dts/rk3568-xgp.dts

```diff
@@ -0,0 +1,408 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
```

Comment:
same here, rename, and this dts file should be same as the one in target (except kernel-specific hack props/nodes).